doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -705,6 +730,8 @@ following figure. The optional :token:`scope` is described in the Section 12.2.
                  : | CoFixpoint `cofix_body` [`decl_notation`] with … with `cofix_body` [`decl_notation`].
    decl_notation : [where `string` := `term` [: `scope`] and … and `string` := `term` [: `scope`]].
    modifiers     : at level `natural`
+                 : in custom `ident`
+                 : in custom `ident` at level `natural`
```

Comment:
I have tried to remove most uses of `natural` and replace them with `num` but apparently failed to do so in this grammar.
Note that using `num` would put a link to https://coq.inria.fr/refman/language/gallina-specification-language.html#grammar-token-num while using `natural` would put a link to https://coq.inria.fr/refman/proof-engine/ssreflect-proof-language.html#grammar-token-natural (`natural` is defined as `num` or `ident` in the SSR chapter).

Reply:
OK, I made the change.